celestia.iss

```diff
@@ -69,21 +64,33 @@ Source: "start.cel";    DestDir: "{app}"; Flags: ignoreversion
 Source: "celestia.cfg"; DestDir: "{app}"; Flags: ignoreversion
 Source: "demo.cel";     DestDir: "{app}"; Flags: ignoreversion
 Source: "guide.cel";    DestDir: "{app}"; Flags: ignoreversion
-Source: "intl.dll";     DestDir: "{app}"; Flags: ignoreversion
-Source: "iconv.dll";    DestDir: "{app}"; Flags: ignoreversion
-Source: "lua5.1.dll";   DestDir: "{app}"; Flags: ignoreversion
+Source: "avcodec-58.dll";     DestDir: "{app}"; Flags: ignoreversion

```

Comment:
is it possible to use just `*.dll`? filenames may change breaking the installer

Reply:
yeah let me try wildcard for all files, they should have been organized already by cmake.